test/parallel/test-util-promise-hooks.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const { disableCrashOnUnhandledRejection } = require('../common');
+disableCrashOnUnhandledRejection();
+process.on('unhandledRejection', () => {});
+
```

Comment:
Can you please add a test for async/await functions? In the past we had a couple of regressions on those.